.github/workflows/test.yml

```diff
@@ -15,12 +15,12 @@ jobs:
       matrix:
         python:
           - "3.8"
```

Comment:
basically everything gets tested with python 3.8 because it's the base python in focal -- the oldest supported LTS.  Next April, we can drop support for python 3.8 and focus only on 3.10 and 3.12.  But for now -- this is the lowest common denominator where our products need to work

Reply:
alright, thanks for the context!